### DISCUSSION TITLE
Run a python code on a node terminal

### DISCUSSION THREAD

**Discussion by kotianshivani** [coreemu/core#817] [Q&A]
Hi everyone, I'm running Core on a virtual box on my Windows laptop. I'm trying to run a knn python script on one of the node terminals but it gives the following error. I've tried pip install pandas in this nodes terminal but it doesnt install it. I already have pandas in my system but it still doesnt work. If anyone has worked on this before please help me out. Thank you!
![image](https://github.com/coreemu/core/assets/31771602/4648aec5-a9f7-466c-9b53-3de5df153fe4)

**Comment by bharnden** [comment]
Not sure what you mean by, it doesn't work when you install it. But, at this point installing your dependency to run a python script is not a CORE issue.

CORE does not manage the system, python, or how you install things.
